packages/react-provider/src/reducer.ts

```diff
@@ -33,6 +33,13 @@ const rootReducer = (state, action) => {
         transport: action.payload.transport,
         userId: action.payload.userId,
         userSignature: action.payload.userSignature,
+        brandId: action.payload.brandId,
+      };
+    }
+    case "root/GET_BRAND/DONE": {
```

Comment:
what do we do while we are fetching a brand?  we prob don't want to render anything right?